lib/services/groups/groupRegistryMongoDB.js

```diff
@@ -117,6 +117,45 @@ function createGroup(group, callback) {
     });
 }
 
+/**
+ * List all the configGroups created in the IoT Agent.
+ *
+ * @param {Number} service      Service for wich all the configurations want to be retrieved.
+ * @param {Number} limit        Maximum number of entries to return.
+ * @param {Number} offset       Number of entries to skip for pagination.
+ */
+function listConfigGroups(service, limit, offset, callback) {
```

Comment:
This function seems to be the same than `listGroups()`. Could you elaborate on the differences, please?

Reply:
In the `listGroups()` function, the callback function stores the result into `services` element while in `listConfigGroups()` function the callback function stores the result into `configGroups` element. It is difficult to change the returned element on fly mode as it is not known which endpoint calls the `listGroups()` inside the function. So, I have created two different functions for /iot/services endpoint and /iot/configGroups endpoint. If my understanding is not correct, please share your opinion.